Mage.Sets/src/mage/cards/t/TARDIS.java

```diff
@@ -0,0 +1,67 @@
+package mage.cards.t;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.CastOnlyIfConditionIsTrueAbility;
+import mage.abilities.condition.Condition;
+import mage.abilities.condition.common.PermanentsOnTheBattlefieldCondition;
+import mage.abilities.effects.common.PlaneswalkEffect;
+import mage.abilities.effects.common.continuous.NextSpellCastHasAbilityEffect;
+import mage.abilities.hint.ConditionHint;
+import mage.abilities.hint.Hint;
+import mage.abilities.keyword.CascadeAbility;
+import mage.constants.ComparisonType;
+import mage.constants.SubType;
+import mage.abilities.keyword.FlyingAbility;
+import mage.abilities.keyword.CrewAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.abilities.common.AttacksTriggeredAbility;
+import mage.abilities.decorator.ConditionalInterveningIfTriggeredAbility;
+import mage.filter.common.FilterControlledPermanent;
+
+/**
+ *
+ * @author Skiwkr
+ */
+public final class TARDIS extends CardImpl {
+    private static final FilterControlledPermanent TimeLordFilter = new FilterControlledPermanent();
```

Comment:
naming convention: fields/methods are not capitalized (only classes). Since only one filter can just call it `filter`